From b55c6847b18fdee5a72d601b871d73085481e4d9 Mon Sep 17 00:00:00 2001
From: "Jason M. Bills" <jason.m.bills@linux.intel.com>
Date: Mon, 3 Jun 2019 16:31:29 -0700
Subject: [PATCH] Add chassis power-cycle and reset to Chassis State

This adds chassis PowerCycle and Reset as possible Transition
values for the Chassis State. These are to support chassis
power cycle and a chassis reset button, respectively.

Signed-off-by: Jason M. Bills <jason.m.bills@linux.intel.com>
---
 xyz/openbmc_project/State/Chassis.interface.yaml | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/xyz/openbmc_project/State/Chassis.interface.yaml b/xyz/openbmc_project/State/Chassis.interface.yaml
index 4ae6274..470a3a2 100644
--- a/xyz/openbmc_project/State/Chassis.interface.yaml
+++ b/xyz/openbmc_project/State/Chassis.interface.yaml
@@ -31,10 +31,17 @@ enumerations:
       values:
         - name: 'Off'
           description: >
-            Chassis  power should be off
+            Chassis power should be off
         - name: 'On'
           description: >
             Chassis power should be on
+        - name: 'PowerCycle'
+          description: >
+            Chassis power should be cycled
+        - name: 'Reset'
+          description: >
+            Chassis should be reset
+
 
     - name: PowerState
       description: >
-- 
2.7.4

